src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -322,7 +322,7 @@ static void regclass_graph_InputTensorInfo(py::module m) {
             return &self.set_color_format(format, sub_names);
         },
         py::arg("format"),
-        py::arg("sub_names"));
+        py::arg("sub_names") = std::vector<std::string>{});
```

Comment:
Please look up `help(openvino.preprocess.InputTensorInfo.set_color_format)`.

Reply:
Help on instancemethod in module openvino.pyopenvino.preprocess:

set_color_format(...)
    set_color_format(self: openvino.pyopenvino.preprocess.InputTensorInfo, format: ov::preprocess::ColorFormat, sub_names: List[str] = []) -> openvino.pyopenvino.preprocess.InputTensorInfo

Reply:
This is fine repr `[]`. Might be resolved if the list is not modified during function call.